.github/workflows/pr_tests.yml

```diff
@@ -4,6 +4,14 @@ on:
   pull_request:
     branches:
       - main
+    paths:
```

Comment:
Why include paths?

Reply:
So that we don't trigger this workflow for changes made to "docs" folder for example.